client/src/Components/Tasks/TaskListItem.js

```diff
@@ -13,15 +14,39 @@ const TaskListItem = ({ task }) => {
       style={{ marginTop: "5px" }}
     >
       <FlexColumn>
-        <Checkbox label={task.description} />
+        <FlexRow style={{ alignItems: "baseline" }}>
+          <Checkbox label={task.description} />
```

Comment:
What is the checkbox for? To mark the task complete or select multiple items for an action?